config/custom-environment-variables.json

```diff
@@ -41,7 +41,10 @@
   },
   "storeTrigger": {
     "url": "STORE_TRIGGER_URL",
-    "subUrl": "STORE_TRIGGER_SUB_URL"
+    "subUrl": {
```

Comment:
Remove the suburls. We will use it hard-coded since we have more than one suburl. There is no added value to do this configurabilly

Reply:
I think it should be in the configuration and not hard codded because it is the best practice.

Reply:
It is not. you have many subUrls so you had to refactor the code according to those subUrls. There is no added value in making it configurable.